kombu/transport/redis.py

```diff
@@ -349,9 +349,9 @@ def maybe_check_subclient_health(self):
         for channel in self._channels:
             # only if subclient property is cached
             client = channel.__dict__.get('subclient')
-            if client is not None:
-                if callable(getattr(client, "check_health", None)):
-                    client.check_health()
+            if client is not None \
```

Comment:
Why is this change being made?  I personally find the prior version more readable since both end up spanning two lines.

Reply:
@matteius I don't feel too strong here but I generally avoid nested if statements if they aren't necessary since "flat is better than nested". When I was reading this I found it less readable because of the nest (but maybe thats just me). But I am not a maintainer so it is not up to me 😄.

Reply:
@auvipy Ok I will resolve this conversation then.